crates/jstzd/tests/octez_baker_test.rs

```diff
@@ -55,24 +61,22 @@ async fn test_baker() {
     let _ = octez_node.kill().await;
 }
 
-async fn spawn_octez_node() -> (octez_node::OctezNode, TempDir) {
-    let temp_dir = TempDir::new().unwrap();
-    let data_dir = temp_dir.path();
+async fn spawn_octez_node() -> jstzd::task::octez_node::OctezNode {
```

Comment:
nit import `jstzd::task::octez_node::OctezNode`

Reply:
There's too much potential confusion now with OctezNode in crate jstzd and crate octez. I prefer to keep the fully-qualified name.